robotiq_c_model_control/src/robotiq_c_model_control/robotiq_gripper_action_interface.py

```diff
@@ -236,6 +279,29 @@ def reset_gripper(self):
         self.reset_thread = Thread(target=self.reset, name='Robotiq Gripper Reset Thread')
         self.reset_thread.start()
 
+    def do_calibration_move(self, direction=GripperGoal.OPEN, wait_seconds=1.25):
+        """Do a calibration command.
+
+        Send a calibration command to the gripper and wait for the next status after completion.
+
```

Comment:
I'm not sure we settled on a docstring but I think we are mostly using sphinx so
:params direction: gripper open/close
:type direction: GripperGoal